examples/nlp/token_classification/punctuation_capitalization_infer.py

```diff
@@ -73,8 +92,8 @@
 
```

Comment:
Do we need amp_opt_level in inference files?

Reply:
removed